frontend/src/components/maps/leaflet/InfoSlideOut/InfoContent.test.tsx

```diff
@@ -0,0 +1,250 @@
+  },
+  landArea: 123,
+  landLegalDescription: 'test description',
+  buildings: [],
+  parcels: [],
+  agency: 'AEST',
+} as IParcel;
+
+export const mockBuilding = {
+  name: 'test name',
+  id: 100,
+  parcelId: '',
+  pid: '',
+  address: {
+    id: 1,
+    line1: '1234 mock Street',
+    administrativeArea: 'Victoria',
+    province: 'BC',
+    postal: 'V1V1V1',
+    provinceId: '1',
+  },
+  latitude: 48,
+  longitude: 123,
+  buildingConstructionTypeId: 0,
+  buildingConstructionType: 'Concrete',
+  classificationId: 1,
+  classification: 'Core Operational',
+  assessedLand: 10000,
+  assessedBuilding: 10000,
+  evaluations: [
+    {
+      date: new Date(),
+      key: 'Assessed',
+      value: 10000,
+    },
+  ],
+  fiscals: [
+    {
+      fiscalYear: 2020,
+      key: 'NetBook',
+      value: 10000,
+    },
+  ],
+  rentableArea: 100,
+  totalArea: 200,
+  agency: 'AEST',
+  agencyId: 0,
+  agencyCode: 'BCIT',
+  subAgency: 'BCIT',
+  transferLeaseOnSale: false,
+  isSensitive: false,
+  buildingPredominateUse: 'University/College',
+  buildingPredominateUseId: 0,
+  buildingOccupantTypeId: 0,
+  encumbranceReason: '',
+  occupantName: 'test',
+  parcels: [mockParcelNoSub],
+  buildingTenancy: '100%',
+} as IBuilding;
+
+export const mockParcel = {
+  id: 1,
+  pid: '000-000-000',
+  pin: '',
+  projectNumber: '',
+  zoning: '',
+  zoningPotential: '',
+  classificationId: 1,
+  encumbranceReason: '',
+  agencyId: '',
+  isSensitive: false,
+  latitude: 48,
+  longitude: 123,
+  classification: 'Core Operational',
+  name: 'test name',
+  description: 'test',
+  assessedLand: 10000,
+  assessedBuilding: 10000,
+  evaluations: [
+    {
+      date: new Date(),
+      key: 'Assessed',
+      value: 10000,
+    },
+  ],
+  fiscals: [
+    {
+      fiscalYear: 2020,
+      key: 'NetBook',
+      value: 10000,
+    },
+  ],
+  address: {
+    id: 1,
+    line1: '1234 mock Street',
+    line2: 'N/A',
+    administrativeArea: 'Victoria',
+    province: 'BC',
+    postal: 'V1V1V1',
+    provinceId: '1',
+  },
+  landArea: 123,
+  landLegalDescription: 'test description',
+  buildings: [mockBuilding],
+  parcels: [],
+  agency: 'AEST',
+  subAgency: 'BCIT',
+} as IParcel;
+
+const ContentComponent = (
+  propertyInfo: IParcel | IBuilding | null,
+  propertyTypeId: PropertyTypes | null,
+  canViewDetails: boolean,
+) => {
+  return (
+    <InfoContent
+      propertyInfo={propertyInfo}
+      propertyTypeId={propertyTypeId}
+      canViewDetails={canViewDetails}
+    />
+  );
+};
+
+describe('InfoContent View', () => {
+  it('InfoContent renders correctly', () => {
+    const { container } = render(ContentComponent(mockParcel, PropertyTypes.PARCEL, true));
+    expect(container.firstChild).toMatchSnapshot();
+  });
+
+  it('Shows all parcel information when can view', () => {
+    const { getByText } = render(ContentComponent(mockParcel, PropertyTypes.PARCEL, true));
+    expect(getByText('Parcel Identification')).toBeVisible();
```

Comment:
want to be a little careful with tests like this, because they can be brittle. Still, i think I like the specific asserts better then a snapshot so please leave as is.